base/applications/cmdutils/taskkill/lang/de-DE.rc

```diff
@@ -16,4 +16,6 @@ STRINGTABLE
     STRING_ENUM_FAILED, "Fehler: Prozessliste kann nicht aufgebaut werden.\n"
     STRING_TERMINATE_FAILED, "Fehler: Prozess ""%1"" kann nicht beendet werden.\n"
     STRING_SELF_TERMINATION, "Fehler: Der Prozess kann sich nicht selbst beenden.\n"
+	STRING_PARAM_TOO_MUCH, "Error: Invalid syntax. '%1' option is not allowed more than '%2!d!' time(s).\n"
+	STRING_INVALID_SYNTAX, "Error: Invalid syntax.\n"
```

Comment:
Unnecessary indentation space here.

Reply:
Looks like this has already been fixed, but I wanted to point out that while GitHub's diff makes it look like this is indented further than it's supposed to be, it was actually an issue of using tabs vs spaces.